### DISCUSSION TITLE
Can I use built in eslint features to build an vs code extension

### DISCUSSION THREAD

**Discussion by Kshitiz1111** [microsoft/vscode-discussions#2317] [Extension Development QnA] [answered]
I want to build an vs code extension to highlight unused code of an editor, like if a function or variable is declared but is not being used.

which is already possible in eslint, how can i use that build in feature and customized the styles?

**Comment by imbant** [comment, marked as answer]
I think the language servers use [DiagnosticTag](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnosticTag).Unnecessary to render unused variables faded out.

Unfortunately, this style seems to be set by the client(like VS Code), just like the error squiggle. It's not like the way client render semantic token modifiers. VS Code does not seem to allow users to customize this style
